zest/releaser/pypi.py

```diff
@@ -407,6 +407,31 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string formating the tag name in the release.
+
+        Override the default ``%(version)`` in setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
+            result = self.config.get(
+                'zest.releaser', 'tag-format')
+            try:
+                '%(version)s'%({'version':''})
+            except KeyError:
+                raise ValueError('version needs to be part of the string formater use default')
```

Comment:
I realised here that the `ValueError` will get caught and not reported.
Is it ok to silently fallback on the default if the tag format is missing the version?
Should I `print()` a warning or should we `exit` completely?